src/hubspot-client.csproj

```diff
@@ -16,6 +16,14 @@
     <PackageId>HubSpotClient</PackageId>
     <Company>SKARP ApS</Company>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard2.0|AnyCPU'">
```

Comment:
**question**: I'm not sure we want these changes in the `csproj` file to be part of the PR

Reply:
I didn't mean the csproj change to be part of the PR @nover. I'll remove it.

Reply:
I have reverted the csproj change but I did not use the Visual Studio Revert command because I was afraid it would Revert the entire commit and not just the commit of the csproj file change. I haven't used the GitHub integration of VS before so I am a bit unsure of myself.

Reply:
you're doing great, and this is exactly why a review process is good to have :)